src/main/java/org/javacord/bot/commands/InviteCommand.java

```diff
@@ -0,0 +1,20 @@
+package org.javacord.bot.commands;
+
+import de.btobastian.sdcf4j.Command;
+import de.btobastian.sdcf4j.CommandExecutor;
+import org.javacord.api.entity.channel.TextChannel;
+import org.javacord.api.entity.message.embed.EmbedBuilder;
+import org.javacord.bot.Constants;
+
+public class InviteCommand implements CommandExecutor {
```

Comment:
There should be some JavaDocs for the class, like there are for the other command classes as well.